src/blocks/accordion/accordion-item/edit.js

```diff
@@ -91,7 +91,7 @@ class AccordionItemEdit extends Component {
 							}
 						} }
 					/>
-					<Icon icon={ isEditing === true || attributes.open ? 'arrow-down' : 'arrow-right' } />
+					<Icon style={ { color: textColor.color } } icon={ isEditing === true || attributes.open ? 'arrow-down' : 'arrow-right' } />
```

Comment:
The last piece to add is the color classname. We need the same logic we see In this file on line 77. Within the Icon component we should add `className={ classnames( { 'has-text-color': textColor.color } ) }`